lib/generator._js

```diff
@@ -461,6 +454,9 @@ ScriptGenerator.prototype.generateAspNetCoreScript = function (templateFileName,
   argNotNull(templateFileName, 'templateFileName');
 
   var lowerCaseScriptType = this.scriptType.toLowerCase();
+  var fixedRestore = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.restore) : fixPathSeperatorToUnix(options.restore);
```

Comment:
Thanks. I totally forgot that PowerShell was available on Unix. I will make the change.